domain/src/main/java/com/andrdoidlifelang/domain/repository/ThemeRepository.kt

```diff
@@ -0,0 +1,15 @@
+package com.andrdoidlifelang.domain.repository
+
+import com.andrdoidlifelang.domain.model.Theme
+import kotlinx.coroutines.flow.Flow
+
+interface ThemeRepository {
+
+    fun getListTheme(): Flow<UseCaseResult<List<Theme>>>
+
+    fun setTheme(theme: Theme): UseCaseResult<Unit>
+
+    fun getTheme(): Flow<UseCaseResult<Theme>>
+
+    fun getThemeObservable(): Flow<UseCaseResult<Theme>>
```

Comment:
Rename to `getTheme()` vs `getThemeObservable()` khác gì nhau vậy

Reply:
getTheme chỉ get tại thời điểm mình muốn get ra. 

getThemeObservable thì khi thằng theme thay đổi trong SharePref thì nó sẽ bắn observer cho những thằng lắng nghe nó.
Cụ thể là MainActivity lắng nghe Observer.